Newsletters/Yearn Finance Newsletter #48/Yearn Finance Newsletter #48.md

```diff
@@ -1,135 +1,133 @@
 
-Got questions or want to learn more? Join us on Telegram or [Discord](https://discord.com/invite/SNPvCpjRET).
+¿Tienes preguntas o quieres aprender más? Únete a nuestro Telegram o [Discord](https://discord.com/invite/SNPvCpjRET).
 
-## Yearn Reaches $5B TVL (Again)
+## Yearn alcanza 5 Billones de dólares en TVL (De nuevo)
 
  ![](image3.png)
 
-Total value locked across all Yearn products has surpassed $5 billion once again, and $1 billion has been added in the last month alone. Yearn is continuing to see parabolic growth among its products with no indication of slowing down. The Iron Bank and v2 vaults are particularly notable components of our exponential growth. You can view and track TVL metrics among our products using [yearn.science](https://yearn.science/). 
+El valor total bloqueado (TVL) a través de todo Yearn ha superado los 5 billones de dólares una vez más y 1 ilion ha sido obtenido en el mes pasado. Yearn continúa viendo crecimiento parabólico en sus productos sin mostrar signos de disminuir el paso. El Iron Bank y la versión V2 son componentes fundamentales de nuestro crecimiento exponencial. Puedes ver y monitorizar las métricas del TVL en todos nuestros productos utilizando [yearn.science](https://yearn.science/). 
 
-A big thank you to everyone in the Yearn ecosystem, our partners, and of course you, the users.
+Muchas gracias a todos los participantes del ecosistema de Yearn, nuestros colaboradores y por supuesto a los usuarios.
 
-## How You Can Contribute to Yearn
+## ¿Cómo puedes colaborar con Yearn?
 
  ![](image4.png)
 
-You may ask, “How can I contribute to Yearn?” For one, Yearn has no papers of incorporation, no headquarters, nor even a list of names and locations for its contributors. It’s a radical experiment in decentralization & here’s how easy it is to get involved.
+Te preguntarás, ¿cómo puedo colaborar con Yearn? Para empezar, Yearn no posee papeles de incorporación, no hay headquaters y ni siquiera una lista de nombres de locaciones para sus contribuidores. En un experimento radical en la descentralización y así es como puedes involucrarte.
 
-Visit our public page at [yearnfinance.notion.site](https://yearnfinance.notion.site). Here, you’ll learn about Yearn’s decentralized structure, culture of doing, open roles, and how to get in touch.
+Visita nuestra página pública en [yearnfinance.notion.site](https://yearnfinance.notion.site). Aquí aprenderás sobre la estructura descentralizada de Yearn, la cultura del hacer, roles disponibles y como ponerte en contacto.
 
-Looking to jump in now and start doing? 1. Visit [contribute.yearn.rocks](https://contribute.yearn.rocks) 2. Scan the list of available tasks 3. Do one that you want to do!
+¿Buscas empezar ahora y empezar a colaborar? 1.- Visita [contribute.yearn.rocks](https://contribute.yearn.rocks) 2.- Revisa la lista de tareas disponibles 3.- ¡Haz alguna que te interese!
```

Comment:
AS-IS: ¿Buscas empezar ahora y empezar a colaborar? 1.- Visita [contribute.yearn.rocks](https://contribute.yearn.rocks) 2.- Revisa la lista de tareas disponibles 3.- ¡Haz alguna que te interese!

TO-BE: ¿Quieres empezar a colaborar? 1.- Visita [contribute.yearn.rocks](https://contribute.yearn.rocks) 2.- Revisa la lista de tareas disponibles 3.- ¡Haz alguna que te interese!